object-construction-checker/tests/socket/ACSocketTest.java

```diff
@@ -479,7 +477,13 @@ private SSLSocket configureSSLSocket(@Owning SSLSocket socket, boolean isClientS
 
 
     private void updateSocketAddresses( SelectionKey sockKey ) {
-        // error: required.method.not.called
+        Socket socket = ((SocketChannel) sockKey.channel()).socket();
+        SocketAddress localSocketAddress = socket.getLocalSocketAddress();
+        SocketAddress remoteSocketAddress = socket.getRemoteSocketAddress();
+    }
+
+    private void updateSocketAddressesWithOwning(@Owning SelectionKey sockKey ) {
+        // TODO we should report a warning here
```

Comment:
Why don't we report a warning now?  Please explain

Reply:
SelectionKey doesn't implement Closeable, so we can't handle @Owning annotation on that. I commented out this test case.